metabase/metabase
# New Metric page (`/metric/query`)

Closes #37353

----


https://github.com/metabase/metabase/assets/6830683/7c98d89e-4c26-4529-97e7-0482bed8e994



### How to verify

1. "+ New" > "Metric"
2. Choose any data source
3. Choose any query shape (enforcing correct Metric query shape will be done in #37355)
4. Visualize the query - URL should still be `/metric/*`
5. Save - you should be able to provide description & choose any parent Collection
    - :warning:  Make sure to include the word "metric" in the metric name so that our mocks kick in
6. You should be redirected to `/metric/:slug` page (this page does not exist yet, it's being added in #37368)
